models/nc-news.models.js

```diff
@@ -13,42 +13,62 @@ exports.fetchTopics = () => {
 };
 
 exports.fetchArticles = (articleId, topicQuery) => {
-  let queryStr = `
-  SELECT articles.article_id, articles.title, users.name AS author,`;
+  return db
+    .query(`SELECT * FROM topics`)
+    .then(({ rows }) => {
+      if (topicQuery) {
+        const activeTopics = [];
+        rows.forEach((row) => {
```

Comment:
Just a small one, but if you wanted to clean this up, a .map would be a better option as it already gives you an array.